main/src/library/MutList.flix

```diff
@@ -90,6 +90,11 @@ mod MutList {
         let MutList(_, _, l) = v;
         deref l
 
+    ///
+    /// Returns the number of elements in the given mutable list `v`.
```

Comment:
currently in? (same for other mutable things?)

Reply:
For the userfacing api there isn't anything else it could be, so it feels redundant